livre/core/Render/GLContext.h

```diff
@@ -43,6 +43,8 @@ class GLContext : public boost::enable_shared_from_this< GLContext >
      */
     void shareContext( GLContextPtr dstContextPtr );
 
+    const GLEWContext* glewGetContext() const;
```

Comment:
Very stupid name selection then :) Thats OK then.

All GL classes are in relationship with GLEW now, then thats ok. Do you know how 3rd party libraries behave under glew ?